### DISCUSSION TITLE
The video download of Dash format does not show anything

### DISCUSSION THREAD

**Discussion by Raptor3355** [aclap-dev/video-downloadhelper#243] [Help and Q&A]
### Your question

Is this because of DMR protection?
Are the Dash mode download the mistakes?

### Addon & CoApp version and details

Version 8.2.1.1

CoApp 2.0.19

**Comment by GrampaWildWilly** [comment]
OnlyFans . . .  There have been some reports in here that OnlyFans has started protecting their content with DRM.  VDH has never, does not now, nor will it ever download content protected by Digital Rights Management.  DRM exists so that content creators can prevent pirating of their copyrighted material.  VDH respects that.  I have recently also done some digging & it seems many screen recorders also now refuse to record content protected by DRM.  OBS used to work fine with DRM content but I've just gotten a new release of that & it doesn't work on DRM protected content.  The image you show above is typical of what VDH will download if the content is protected by DRM.  It has nothing to do with HLS vs DASH.  It's protected by DRM.  You are out of luck.

**Reply by drbit786** [reply]
That's helpful. Thanks. I find that those same files which I can play on on my desktop (Windows) but can't be recorded wth VDH don't even play on my iphone. Any thoughts on how to overcome this prroblem?
